src/api/app/models/staging_workflow.rb

```diff
@@ -20,10 +20,12 @@ def ignored_requests
 
   private
 
-  def init_staging_projects
-    return unless new_record?
+  def create_staging_projects
     ['A', 'B'].each do |letter|
-      staging_projects << Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
+      p = Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
```

Comment:
NOOOOO!!!

`p` is for printing in ruby... please use something else :smile: 

What about `staging_project`?